### PR TITLE
Make the README.md more useful in links to man page documentation

### PR DESCRIPTION
**Pull request by rhatdan**
This patch will allow people who go to the github page to easily read
man pages.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
Looks good

File: README.md [hunk 1]
```diff
@@ -36,6 +36,22 @@ The plan is to use OCI projects and best of breed libraries for different aspect
 
 It is currently in active development in the Kubernetes community through the [design proposal](https://github.com/kubernetes/kubernetes/pull/26788).  Questions and issues should be raised in the Kubernetes [sig-node Slack channel](https://kubernetes.slack.com/archives/sig-node).
 
+## Commands
+| Command                                              | Description                                                                                          |
+| ---------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [crio(8)](/docs/crio.8.md)                 | Enable OCI Kubernetes Container Runtime daemon |
+| [kpod(1)](/docs/kpod.1.md)                 | Simple management tool for pods and images |
```
Comment (TomSweeneyRedHat) [inline README.md]: The page that gets pulled up says kpod(8)  should this be changed to 8 or should the page change to 1?  I think the page should change to 1.

File: README.md [hunk 2]
```diff
@@ -36,6 +36,22 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
+| [kpod-images(1)](/docs/kpod-images.1.md)   | List images in local storage. |
+[kpod-pull(1)](/docs/kpod-pull.1.md)         | Pull an image from a registry |
```
Comment (TomSweeneyRedHat) [inline README.md]: kpod-pull's page also says 8 and probably should be 1.

File: README.md [hunk 3]
```diff
@@ -36,6 +36,22 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
+[kpod-rmi(1)](/docs/kpod-rmi.1.md)           | Removes one or more images.   |
+[kpod-tag(1)](/docs/kpod-tag.1.md)           | Add an additional name to a local image. |
+[kpod-version(1)](/docs/kpod-version.1.md)   | Display the Kpod Version Information |
```
Comment (TomSweeneyRedHat) [inline README.md]: kpod-version's page also claims 8.

File: README.md [hunk 4]
```diff
@@ -36,6 +36,22 @@ The plan is to use OCI projects and best of breed libraries for different aspect
 
 It is currently in active development in the Kubernetes community through the [design proposal](https://github.com/kubernetes/kubernetes/pull/26788).  Questions and issues should be raised in the Kubernetes [sig-node Slack channel](https://kubernetes.slack.com/archives/sig-node).
 
+## Commands
+| Command                                              | Description                                                                                          |
+| ---------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [crio(8)](/docs/crio.8.md)                 | Enable OCI Kubernetes Container Runtime daemon |
```
Comment (TomSweeneyRedHat) [inline README.md]: The first line of each of these pages are not consistent and should be.  Some are bold, some have dates, some have authors, others do not.  Also some have their number a "1" or "8" and it should be (1) or (8).  The changes here in this particular page are fine, so I'm good with getting this submitted and creating a follow up/issue to make these called pages consistent.

File: docs/crio.conf.5.md [hunk 5]
```diff
@@ -3,10 +3,10 @@
 % OCTOBER 2016
 
 # NAME
-crio.conf - Syntax of CRIO configuration file
+crio.conf - CRI-O configuration file
```
Comment (TomSweeneyRedHat) [inline docs/crio.conf.5.md]: 👍

File: docs/kpod-images.1.md [hunk 6]
```diff
@@ -1,15 +1,17 @@
+% kpod(1) kpod-images - List images in local storage.
+% Dan Walsh
 ## kpod-images "1" "March 2017" "kpod"
 
-## NAME
-kpod images - List images in local storage.
+# NAME
+kpod images -
```
Comment (TomSweeneyRedHat) [inline docs/kpod-images.1.md]: I'd put the "List images in local storage." back here, but if you think it's best to be left off, please remove the space and '-' at the end of the line

Comment (rhatdan) [inline docs/kpod-images.1.md]: That looks like a mistake, I will fix.

Comment (rhatdan) [inline docs/kpod-images.1.md]: Fixed

File: README.md (hunk 4 above)
Comment (TomSweeneyRedHat) [inline README.md]: Do you want to remove 'Enable' here too?

File: README.md [hunk 7]
```diff
@@ -36,6 +36,22 @@ The plan is to use OCI projects and best of breed libraries for different aspect
...
+| [kpod-images(1)](/docs/kpod-images.1.md)   | List images in local storage. |
+[kpod-pull(1)](/docs/kpod-pull.1.md)         | Pull an image from a registry |
+[kpod-rmi(1)](/docs/kpod-rmi.1.md)           | Removes one or more images.   |
+[kpod-tag(1)](/docs/kpod-tag.1.md)           | Add an additional name to a local image. |
+[kpod-version(1)](/docs/kpod-version.1.md)   | Display the Kpod Version Information |
+
+## Configuration
+| File                                              | Description                                                                                          |
+| ---------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| [crio.conf(5)](/docs/crio.conf.5.md)         | Crio Configuation file |
```
Comment (TomSweeneyRedHat) [inline README.md]: To match crio.confg.5.md
'CRI-O configuration file'

File: docs/kpod-tag.1.md [hunk 8]
```diff
@@ -1,29 +1,34 @@
-## kpod-tag "1" "June 2017" "kpod"
+% kpod(1) kpod-tag - add tags to an image
+% Ryan Cole
+# kpod-tag "1" "July 2017" "kpod"
 
-# NAME
+## NAME
 kpod tag - add tags to an image
```
Comment (TomSweeneyRedHat) [inline docs/kpod-tag.1.md]: I'd change this line to match the README.md

kpod tag - Add an additional name to a local image

File: README.md (hunk 4 above)
Comment (TomSweeneyRedHat) [inline README.md]: Just noticed, some of the descriptions end with a period, others do not.  I think I'd remove the period to match the man page variant.

File: docs/kpod-tag.1.md (hunk 8 above)
Comment (rhatdan) [inline docs/kpod-tag.1.md]: Fixed

File: README.md (hunk 7 above)
Comment (rhatdan) [inline README.md]: Fixed

File: docs/kpod.1.md [hunk 9]
```diff
@@ -19,27 +19,27 @@ has the capability to debug pods/images created by crio.
...
   Print usage statement
 
 **--version, -v**
   Print the version
 
-# COMMANDS
+## COMMANDS
 
-## images
+### images
```
Comment (TomSweeneyRedHat) [inline docs/kpod.1.md]: should this be 2 or 3 #'s?  Ditto line 35.

File: README.md (hunk 4 above)
Comment (rhatdan) [inline README.md]: Fixed

File: docs/kpod.1.md (hunk 9 above)
Comment (rhatdan) [inline docs/kpod.1.md]: These are subbullets under COMMAND so they should be the next level of heading.

Comment (rhatdan) [inline docs/kpod.1.md]: Does not effect man page only display on website.

File: docs/kpod.1.md [hunk 10]
```diff
@@ -1,10 +1,10 @@
-% kpod(8) # kpod - Simple management tool for pods and images
+% kpod(1) kpod - Simple management tool for pods and images
 % Dan Walsh
-% SEPTEMBER 2016
-# NAME
+# kpod "1" "September 2016" "kpod"
+## NAME
 kpod
```
Comment (TomSweeneyRedHat) [inline docs/kpod.1.md]: No synopsis/description here?  Maybe:

kpod - Simple management tool for pods and images

File: README.md (hunk 4 above)
Comment (rhatdan) [inline README.md]: Yes, This is a container runtime daemon, not enabling a container runtime daemon.

File: README.md (hunk 7 above)
Comment (TomSweeneyRedHat) [inline README.md]: To match the man, please change the description to:

CRI-O configuration file

File: docs/kpod.1.md (hunk 10 above)
Comment (rhatdan) [inline docs/kpod.1.md]: Added
kpod - Simple management tool for containers and images

File: docs/kpod-images.1.md [hunk 11]
```diff
@@ -1,4 +1,6 @@
-## kpod-images "1" "March 2017" "kpod"
+% kpod(1) kpod-images - List images in local storage.
+% Dan Walsh
+# kpod-images "1" "March 2017" "kpod"
 
 ## NAME
 kpod images - List images in local storage.
```
Comment (TomSweeneyRedHat) [inline docs/kpod-images.1.md]: I'd suggest removing period from end here and at line 1.

File: docs/kpod-rmi.1.md [hunk 12]
```diff
@@ -1,4 +1,6 @@
-## kpod-rmi "1" "March 2017" "kpod"
+% kpod(1) kpod-rmi - Removes one or more images
+% Dan Walsh
+# kpod-rmi "1" "March 2017" "kpod"
 
 ## NAME
 kpod rmi - Removes one or more images.
```
Comment (TomSweeneyRedHat) [inline docs/kpod-rmi.1.md]: I'd suggest yanking ending period here.

File: docs/kpod-tag.1.md (hunk 8 above)
Comment (TomSweeneyRedHat) [inline docs/kpod-tag.1.md]: Sorry if a duplicate, thought I'd made this comment and don't see it.  Please change to:

kpod tag - Add an additional name to a local image

To sync up with the README.md's variant.

**Comment by TomSweeneyRedHat** [comment]
FWIW, I'm done picking.

**Comment by rhatdan** [comment]
@runcom @mrunalp Lets get this merged so we can get other pull requests to update README.md.

**Comment by mrunalp** [comment]
will merge once travis returns.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**